wallet/udb/txdb.go

```diff
@@ -160,6 +160,8 @@ var (
 	rootHaveCFilters = []byte("havecfilters")
 	rootLastTxsBlock = []byte("lasttxsblock")
 	rootVSPHostIndex = []byte("vsphostindex")
+	rootBirthdayTime = []byte("birthdaytime")
+	rootBirthday     = []byte("birthday")
```

Comment:
Did you want "birthblock" here @matheusd ?

Reply:
Renamed Birthday to Birthblock anyway since it was rather close to birthdaytime anyhow and they may get confused.

Reply:
https://github.com/decred/dcrwallet/compare/80145bce22dd5604c93e2c6e25cd5c72a0063885..aa31212a2864f26125d8c8a9e4db4ae91b17e836